src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -205,128 +186,119 @@
+(defn reply-message-view [{:keys [photo-path] :as message}]
+  (when message
+    [react/view {:style styles/reply-wrapper}
+     [react/view {:style styles/reply-container}
+      [react/image {:source {:uri photo-path}
+                    :style  styles/reply-photo-style}]
+      [reply-message message]]
+     [react/touchable-highlight
+      {:style               styles/reply-close-highlight
+       :on-press            #(re-frame/dispatch [:chat.ui/cancel-message-reply])
+       :accessibility-label :cancel-message-reply}
+      [react/view {}
+       [icons/icon :icons/close {:style styles/reply-close-icon}]]]]))
 
-(views/defview chat-text-input [chat-id input-text]
-  (views/letsubs [inp-ref (atom nil)
-                  network-status [:network-status]]
-    {:component-will-update
-     (fn [e [_ new-chat-id new-input-text]]
-       (let [[_ old-chat-id] (.. e -props -argv)]
-         (when (not= old-chat-id new-chat-id)
-           ;; reset default text when switch to another chat
-           (.setNativeProps @inp-ref #js {:text (or new-input-text "")}))))}
-    (let [component               (reagent/current-component)
-          set-container-height-fn #(reagent/set-state component {:container-height %})
-          {:keys [container-height]} (reagent/state component)]
-      [react/view {:style (styles/chat-box container-height)}
-       [react/text-input {:placeholder            (i18n/label :t/type-a-message)
-                          :auto-focus             true
-                          :multiline              true
-                          :blur-on-submit         true
-                          :style                  (styles/chat-text-input container-height)
-                          :font                   :default
-                          :ref                    #(reset! inp-ref %)
-                          :default-value          input-text
-                          :on-content-size-change #(set-container-height-fn (.-height (.-contentSize (.-nativeEvent %))))
-                          :submit-shortcut        {:key "Enter"}
-                          :on-submit-editing      #(when (= :online network-status)
-                                                     (.clear @inp-ref)
-                                                     (.focus @inp-ref)
-                                                     (re-frame/dispatch [:chat.ui/send-current-message]))
-                          :on-change              (fn [e]
-                                                    (let [native-event (.-nativeEvent e)
-                                                          text         (.-text native-event)]
-                                                      (re-frame/dispatch [:chat.ui/set-chat-input-text text])))}]
-       [send-button inp-ref network-status]])))
+(defn chat-text-input [chat-id input-text]
+  (let [inp-ref (atom nil)
+        network-status [:network-status]]
```

Comment:
shouldn't  `network-status` be reagent subscription? now it looks like as const vector `[:network-status]`